wkcuber/wkcuber/cubing.py

```diff
@@ -165,8 +161,7 @@ def cubing_job(
     ]
 ) -> None:
     (
-        target_wkw_info,
-        z_batches,
```

Comment:
my understanding is that `z_batches` was actually a list of individual “dense” `z` values (so basically just one batch) – is that correct? In this case, replacing it by a single view seems fine by me, but maybe there was some specific reason behind `z_batches`? Also compare the old comment at line 184

Reply:
Yes, `z_batches` was a list of "dense" `z`: (`z_batch = list(range(z, max_z))`)